lib/index.js

```diff
@@ -9700,8 +9700,9 @@ async function run() {
 }
 
 function continueFileExists() {
-  const continuePath = process.platform !== "win32" ? "/continue" : "C:/msys64/continue"
-  return external_fs_default().existsSync(continuePath) || external_fs_default().existsSync(external_path_default().join(process.env.GITHUB_WORKSPACE, "continue"))
+  const rootDirContinuePath = process.platform === "win32" ? "C:/msys64/continue" : "/continue"
+  const homeContinueDir = process.platform === "win32" ? "C:/msys64/home/runneradmin" : process.env.GITHUB_WORKSPACE
```

Comment:
Does the path really have to be hard-coded for Windows? Can't it be the same `$GITHUB_WORKSPACE`?

Also: we probably want to extend the documentation to talk about the second location.

Reply:
GITHUB_WORKSPACE won't work, since it's the host (win) directory and not inside this linux shell environment.
I think the documentation is enough.
> just create a empty file with the name continue either in the root directory or in the project directory by running touch continue or sudo touch /continue.

We can use the current username instead of runneradmin, but the rest needs to stay I guess. I can create a small helper function to build these file locations, what would be cleaner.

Reply:
> GITHUB_WORKSPACE won't work, since it's the host (win) directory and not inside this linux shell environment.

Oh, MSYS2 is not a Linux environment, and the initial directory _is_ the workspace, even if it is outside the pseudo root. The path is then of the form `/c/agent/...`.

I really would prefer consistency between Windows and other platforms.

Reply:
Good point! You brang me to the idea, that the PWD on Windows was not correctly set, it was previously `/home/runneradmin` and is now as for Linux and MacOS too the GitHub workspace directory. This also simplifies the check if the continue file exists.

![image](https://user-images.githubusercontent.com/17984549/107142943-72309580-6932-11eb-8974-abbb7063a506.png)